pkg/service/credential/model.go

```diff
@@ -85,8 +85,22 @@ type UpdateCredentialStatusRequest struct {
 }
 
 type UpdateCredentialStatusResponse struct {
-	Revoked   bool `json:"revoked" validate:"required"`
-	Suspended bool `json:"suspended" validate:"required"`
+	Status
+}
+
+type Status struct {
+	// ID of the credentials who's status this object represents.
+	ID        string `json:"id,omitempty"`
```

Comment:
not required?

Reply:
No, the usage in `UpdateCredentialStatusResponse` is optional.